hot_osm/templates/components/branded_elements/link.html

```diff
@@ -0,0 +1,7 @@
+<a href="{{ link }}"
+   class="hover:underline block font-medium font-barlow text-hot-black {{ classes }}">
+    {{ text }}
+    {% if parent|default:False %}
+        <span class="absolute translate-y-[-15%] translate-x-1">⌄</span>
```

Comment:
Don't use absolute here, use padding instead - also with the translate, you can make it negative in tailwind by appending a `-` at the start 👍